openapi.yaml

```
@@ -4084,6 +4317,220 @@ paths:
+                  description: 'Private key UUID'
+                enable_ipv4:
+                  type: boolean
+                  example: true
+                  description: 'Enable IPv4 (default: true)'
+                enable_ipv6:
+                  type: boolean
+                  example: true
+                  description: 'Enable IPv6 (default: true)'
+                hetzner_ssh_key_ids:
+                  type: array
+                  items: { type: integer }
```

YAMLlint (1.37.1)

[error] too many spaces inside braces

(braces)